apps/frontend/src/libs/types/input-options.type.ts

```diff
@@ -0,0 +1,6 @@
+type Options = {
```

Comment:
```suggestion
type RadioInputOption = {
```
Change file name as well

Reply:
Can it be InputOption? I plan to use this for chackbox input.